### ISSUE TITLE
Provide a printable output of the schedule.xml

### ISSUE THREAD

**Issue by thomic** [pretalx/pretalx#504]
## Problem you are facing

Many smaller conferences can't afford high-tech digital signage solutions, may it be due to lack of displays or due to lack of infobeamers and Raspberry Pi's available.
For those conferences - a printable output (like PDF) would be amazing. At the moment - with every browser I tried [Firefox, Chromium] - I wasn't able to get a nice formatted output (like split by day on DIN A4 to just "hang it on the wall").

Example see here: https://cfp.owncloud.com/occon18/schedule/

If you try to print it - it doesn't size well. 

## Possible Solution

Provide a printable output.

## Context

see above.

Thanks for your amazing work! This is an awesome tool.

**Comment by luto** [comment]
```css
@media print {
  body {
    background: none;
    overflow: visible;
  }
  #navigator, #schedule-nav, #main-card .user-row, #event-nonpublic {
    display: none;
  }
  .container {
    max-width: 100%;
  }
  .day {
    break-after: page;
  }
  /* hacks to make page break work */
  h3:not(:first-of-type) {
    margin-top: 5em;
  }
  #fahrplan {
    display: block; 
  }
}
```

A quick CSS to make the page more printable. Page breaks after certain elements (days in this case) do not seem to be supported well in browsers. Our flexbox layout breaks it.. m) Also note that this only seems to work in chrome. Firefox just prints the first day and then stops.

Once the fix for #505 is released, you can upload this in your conference settings. Except for the page break hack, we should also be able to ship this in the future.

**Comment by rixx** [comment]
Printing on Firefox still seems broken, and I can't go and debug it, because https://bugzilla.mozilla.org/show_bug.cgi?id=1202711

Printing on Chromium works, though, so it's a best effort fix for now.
